core/cloudflow-spark-testkit/src/main/scala/cloudflow/spark/testkit/TestSparkStreamletContext.scala

```diff
@@ -60,6 +59,7 @@ private[testkit] class TestSparkStreamletContext(override val streamletRef: Stri
         stream.writeStream
           .outputMode(outputMode)
           .format("memory")
+          .trigger(Trigger.Once)
```

Comment:
Is it way slower if we omit this?

Reply:
The issue I'm observing with the failing test is that there's no data produced for the first query. Looks like the `rate` producer generates the first batch with zero records :-(

Reply:
processAllAvailable waits until you get all the data.